### PR TITLE
Video v2 main image

### PR DESCRIPTION
**Pull request by ironsidevsquincy**
Will now come through as `MediaResource` content, so update the transform accordingly

Also added tests for the other video v2 helpers attachments and storyPackage

### CODE REVIEW

File: server/models/video-v2.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const resolveMetadata = (annotations = [], dependencies = {}) => {
 };
 
 module.exports = ({ capiV2 = {} } = {}, dependencies = {}) => {
-	const deps = Object.assign({ concord, imagesetV2, storyPackage }, dependencies);
+	const deps = Object.assign({ concord, mainImage, storyPackage }, dependencies);
```
Comment (i-like-robots) [inline server/models/video-v2.js]: Just a heads up that this app tends to use proxyquire rather than DI

File: server/models/video-v2.js [hunk 2]
```diff
@@ -2,8 +2,8 @@ const Model = require('./model');
 const metadataV1 = require('./article/metadata-v1');
 const annotations = require('./article/annotations');
 const normalizeV2 = require('./article/normalize-v2');
-const imagesetV2 = require('./article/imageset-v2');
 const attachments = require('./video/attachments');
+const mainImage = require('./video/main-image');
```
Comment (i-like-robots) [inline server/models/video-v2.js]: looks like this functionality already exists (just a bit unexpectedly named) with only slight differences. Maybe worth checking out the two can be reconciled?

https://github.com/Financial-Times/next-es-interface/blob/master/server/models/article/image-v2.js

File: server/models/video-v2.js (hunk 1 above)
Comment (ironsidevsquincy) [inline server/models/video-v2.js]: proxyquire is way too much like magic, but can switch over if you prefer

File: server/models/video-v2.js (hunk 2 above)
Comment (ironsidevsquincy) [inline server/models/video-v2.js]: Ah, will use that instead

File: server/models/video-v2.js (hunk 1 above)
Comment (i-like-robots) [inline server/models/video-v2.js]: I'm not really fussy whatever works for you

**Pull request merged**